configure

```diff
@@ -288,6 +288,11 @@ parser.add_option('--enable-d8',
     dest='enable_d8',
     help=optparse.SUPPRESS_HELP)  # Unsupported, undocumented.
 
+parser.add_option('--enable-trace-maps',
+    action='store_true',
+    dest='trace_maps',
+    help='Enable the --trace-maps flag in v8 (use at your own risk)')
```

Comment:
Actually, one nit: `V8` with a capital V, since we started disambiguating due to Node (v)8 ;)